### PR TITLE
Update unsupported error and support form modal dialogs

### PR DESCRIPTION
**Pull request by bvaughn**
Loom walk through available [here](https://www.loom.com/share/c9121b4d4de349949d9bff90160ddfb9?sid=d657afec-e5c7-477c-bea7-40c0335b7641).

Resolves FE-2246 and FE-2228

cc @jonbell-lot23 

| State | Screenshot |
| :--- | :--- |
| Unexpected error (no details) | Screenshot 2024-02-20 at 5 11 51 PM |
| Unexpected error (details collapsed) | Screenshot 2024-02-20 at 5 11 33 PM |
| Unexpected error (details expanded) | Screenshot 2024-02-20 at 5 15 23 PM |
| Unexpected error (submit a ticket) | Screenshot 2024-02-20 at 5 27 18 PM |
| Unexpected error (Windows recording) | Screenshot 2024-02-20 at 5 27 03 PM |
| Support form | Screenshot 2024-02-20 at 5 27 38 PM |
| Support form (pending edits cancel) | Screenshot 2024-02-20 at 5 11 23 PM |

### CODE REVIEW

File: packages/replay-next/components/Icon.tsx [hunk 1]
```diff
@@ -627,6 +633,10 @@ export default function Icon({
...
+      break;
```
Comment (bvaughn) [inline packages/replay-next/components/Icon.tsx:639]: Technically these icons didn't need to move into this component, but I thought we'd be sharing them between multiple places (before it became clear that Jon wanted these forms to stay nested). This way they'll be easier to reuse if we want to in the future though.

File: packages/replay-next/pages/variables.css [hunk 2]
```diff
@@ -49,7 +49,7 @@
   --primary-accent-hover: #0194ff;
 
   --secondary-accent: #d72451;
-  --secondary-accent-hover: #d72451;
+  --secondary-accent-hover: #c61351;
```
Comment (bvaughn) [inline packages/replay-next/pages/variables.css:52]: This regressed a couple of months ago in #10049

File: packages/replay-next/pages/variables.css [hunk 3]
```diff
@@ -119,6 +119,16 @@
...
+
+  /* The styles below are the same between themes */
+  /* TODO [FE-2246] Are there other light-only styles like this that we should colocate? */
+  --support-form-color: var(--grey-90);
+  --support-form-close-icon-color: #ffffff;
+  --support-form-modal-background-color: #ffffff;
+  --support-form-modal-border-color: var(--grey-90);
+  --support-form-link-background-color: var(--grey-20);
+  --support-form-link-background-color-hover: var(--grey-30);
+  --support-form-text-area-background-color: var(--grey-20);
```
Comment (bvaughn) [inline packages/replay-next/pages/variables.css:131]: I don't _love_ this but I don't want to hard-code these values deep inside of the app either.

File: packages/replay-next/pages/variables.css (hunk 2 above)
Comment (jonbell-lot23) [reply packages/replay-next/pages/variables.css:52]: Thanks for this fix, I was just looking into it because of #10049

**Review by markerikson — APPROVED** [review]

**Pull request merged**